TheWarWithin/DeathKnightFrost.lua

```diff
@@ -15,137 +15,110 @@ local spec = Hekili:NewSpecialization( 251 )
 
 spec:RegisterResource( Enum.PowerType.Runes, {
     rune_regen = {
-        last = function ()
-            return state.query_time
-        end,
+        last = function () return state.query_time end,
+        stop = function( x ) return x == 6 end,
 
         interval = function( time, val )
-            local r = state.runes
-            val = math.floor( val )
-
+            val = floor( val )
```

Comment:
Do they need to be added to each class file? I seem to remember it yelling at me in LUA errors when I used math.floor, and then it stopped yelling and started working when I just used floor like the other classes.

Seems like floor = math.floor is defined more globally since all class files use it without declaring (except 2 specs which declare it for some reason?)

State.lua:669 `state.floor = math.floor`